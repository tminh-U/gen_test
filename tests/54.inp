12
10 2 6 11 3 4 1 14 10 8 2 11 
11
4 4
11 12
11 12
6 12
3 3
1 12
9 11
5 12
2 6
1 11
7 11
