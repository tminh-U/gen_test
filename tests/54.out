2
4
4
25
2
103
4
39
14
60
10
